graph_annotation.go

```diff
@@ -16,3 +25,74 @@ func (c *Client) CreateGraphAnnotation(annotation *GraphAnnotation) error {
 	defer closeResponse(resp)
 	return err
 }
+
+// FindGraphAnnotations fetches graph annotation.
+func (c *Client) FindGraphAnnotations(service string, from int64, to int64) ([]GraphAnnotation, error) {
+	v := url.Values{}
+	v.Add("service", service)
+	v.Add("from", strconv.FormatInt(from, 10))
+	v.Add("to", strconv.FormatInt(to, 10))
+
+	req, err := http.NewRequest("GET", fmt.Sprintf("%s?%s", c.urlFor("/api/v0/graph-annotations").String(), v.Encode()), nil)
+	if err != nil {
+		return nil, err
+	}
+
+	resp, err := c.Request(req)
+	defer closeResponse(resp)
+	if err != nil {
+		return nil, err
+	}
+
+	var data struct {
+		GraphAnnotations []GraphAnnotation `json:"graphAnnotations"`
+	}
+	err = json.NewDecoder(resp.Body).Decode(&data)
+	if err != nil {
+		return nil, err
+	}
+	return data.GraphAnnotations, err
+}
+
+// UpdateGraphAnnotation updates graph annotation.
+func (c *Client) UpdateGraphAnnotation(annotationID string, annotation *GraphAnnotation) (*GraphAnnotation, error) {
+	resp, err := c.PutJSON(fmt.Sprintf("/api/v0/graph-annotations/%s", annotationID), annotation)
+	defer closeResponse(resp)
+	if err != nil {
+		return nil, err
+	}
+
+	var updatedAnnotation = GraphAnnotation{}
+	err = json.NewDecoder(resp.Body).Decode(&updatedAnnotation)
+	if err != nil {
+		return nil, err
+	}
+
+	return &updatedAnnotation, nil
+}
+
+// DeleteGraphAnnotation deletes graph annotation.
+func (c *Client) DeleteGraphAnnotation(annotationID string) (*GraphAnnotation, error) {
+	req, err := http.NewRequest(
+		"DELETE",
+		c.urlFor(fmt.Sprintf("/api/v0/graph-annotations/%s", annotationID)).String(),
+		nil,
+	)
+	if err != nil {
+		return nil, err
+	}
+	req.Header.Add("Content-Type", "application/json")
+
+	resp, err := c.Request(req)
+	defer closeResponse(resp)
+	if err != nil {
+		return nil, err
+	}
+
+	var annotation = GraphAnnotation{}
```

Comment:
[nit] `var annotation GraphAnnotation` is preferred when this line indicates the allocation (not initialization of empty annotation) IMO.